Project.toml

```diff
@@ -36,8 +36,8 @@ DistributionsAD = "0.1.0"
 FiniteDifferences = "0.9"
 ForwardDiff = "0.10.3"
 Libtask = "0.3.1"
+MCMCChains = "^0.3.15"
```

Comment:
The [caret specifiers are the default version specifiers](https://julialang.github.io/Pkg.jl/v1/compatibility/#Version-specifier-format-1), so it's equivalent to writing just "0.3.15" without caret. Maybe you actually wanted to use a tilde specifier?

Reply:
Thanks for catching that, though, I wasn't aware that it was the default version spec.